src/grpcbox_channel.erl

```diff
@@ -57,6 +58,16 @@ start_link(Name, Endpoints, Options) ->
 is_ready(Name) ->
     gen_statem:call(?CHANNEL(Name), is_ready).
 
+-spec get(name(), unary | stream, term()) ->
+    {ok, {pid(), grpcbox_client:interceptor() | undefined}} |
+    {error, undefined_channel | not_found_endpoint}.
+get(Name, CallType, Key) ->
+    case lists:keyfind(Key, 1, gproc_pool:active_workers(Name)) of
+        {_, Pid} -> {ok, {Pid, interceptor(Name, CallType)}};
+        false -> {error, not_found_endpoint}
+    end.
```

Comment:
The design of balance type (strategy) of the original grpcbox is fixed when channel initializing. For an exists channel, the `pick` function can only use the pre-configured strategy. It's better to follow the design IMO.